docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     links:
       - joomladb:mysql
     ports:
-      - "8080:80"
+      - "6001:443"
```

Comment:
Why change the port?  Can't we just have all the examples configured to run on the same port?

Reply:
They will end up colliding if run them simultaneously, that's why I am using a different port for each.

Reply:
ok